CanvasPlusPlayground/Features/Courses/CourseListView.swift

```diff
@@ -216,6 +219,18 @@ private struct CourseListCell: View {
                     course.isFavorite = !wrappedCourseIsFavorite
                 }
             }
+            
+            Button("Add Course Nickname", systemImage: "character.cursor.ibeam") {
+                withAnimation {
+                    showRenameTextField = true
```

Comment:
When you present the alert, we can fill the textfield with the current nickname, if it exists.

Reply:
Are you meaning the textfield value, or the greyed out placeholder text? This is it with the latter

<img width="1012" alt="Screenshot 2024-12-11 at 4 53 22 PM" src="https://github.com/user-attachments/assets/486e153e-80af-4c22-8fe1-2969a7f9ac92" />

Reply:
The placeholder should be the original name, yes. But if you already have a nickname the textfield should be pre-populated with that nickname. So:

```swift
renameCourseFieldText = course.nickname ?? ""
```